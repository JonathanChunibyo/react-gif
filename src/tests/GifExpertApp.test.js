import AppGif from "../GifExpertApp";
import { shallow } from "enzyme";


describe('Probando componente GifExpertApp', () => {

    const category = ['one punch', 'one piece', 'sailor moon'];
    
    test('snapshot de gitExpertApp ', () => {
        
        const wraper = shallow(< AppGif />);
        expect(wraper).toMatchSnapshot();
    })
    test('Inicializar en cero las categorias - poner dos y referenciar cuantas hay', () => {
        
        const wraper = shallow(< AppGif categoriesDefault={category}/>);
        expect(wraper).toMatchSnapshot();
        expect(wraper.find('GifGrid').length).toBe(category.length);
    })
    
    
})
