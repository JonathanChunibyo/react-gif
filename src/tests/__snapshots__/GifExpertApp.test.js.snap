// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probando componente GifExpertApp Inicializar en cero las categorias - poner dos y referenciar cuantas hay 1`] = `
<Fragment>
  <h2>
    Git Expert App
  </h2>
  <AddCategory
    setcategory={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="one punch"
      key="one punch0.48809218924971076"
    />
    <GifGrid
      category="one piece"
      key="one piece0.18737759462217385"
    />
    <GifGrid
      category="sailor moon"
      key="sailor moon0.9398530291482154"
    />
  </ol>
</Fragment>
`;

exports[`Probando componente GifExpertApp snapshot de gitExpertApp  1`] = `
<Fragment>
  <h2>
    Git Expert App
  </h2>
  <AddCategory
    setcategory={[Function]}
  />
  <hr />
  <ol />
</Fragment>
`;
